docs/user-guide/IT_analysis_NSB.rst

```diff
@@ -0,0 +1,39 @@
+.. _IT_data_NSB:
+
+IT Cluster analysis: NSB_matching
+=================================
+
+TODO
+----- 
+
+1. Database (joint observations): choose format and share the needed file
+
+Quick start tutorial
+--------------------
+
+Update the config_h5.yaml file with the time range, target name and bad runs.
+
+``python list_from_h5.py`` to create the lists with the runs
```

Comment:
Here you are describing what each script is doing. No need to have `python` in front, just the name of the script is fine.